Dynavity/Dynavity/view/canvas/elements/ImageCanvasElementView.swift

```diff
@@ -0,0 +1,17 @@
+import SwiftUI
+
+struct ImageCanvasElementView: View {
+    var imageCanvasElement: ImageCanvasElement
```

Comment:
You're right, especially since we will be adding contextual actions to the various canvas elements later on. I'll make the change.

However, if this were just a dumb view (only displaying data from the model with no other functionalities), I'm inclined to not pass the model through a view model. While I acknowledge that this creates a dependency from the view to the model, wrapping the model in a view model (whose sole purpose is to contain the model) feels extremely redundant.

Reply:
I see. Yep, makes sense to me!